custom_components/tuya_local/devices/rectec_rt700_grill.yaml

```diff
@@ -0,0 +1,42 @@
+name: RecTeq RT-700 Grill
+products:
+  - id: 5518137548fda419c2e
+    name: RecTeq RT-700
+primary_entity:
```

Comment:
Should now be `entities`, combining both primary and secondary entities into a single list.